### DISCUSSION TITLE
Accuracy Calculator: How are NMI and AMI calculated?

### DISCUSSION THREAD

**Discussion by HSinger04** [KevinMusgrave/pytorch-metric-learning#595] [Q&A] [answered]
Hello, I have two questions regarding the calculation of the NMI / AMI metrics: I used `tester.test` (where `tester` is an instance of `GlobalEmbeddingSpaceTester`) with `splits_to_eval=[('test', ['samples'])]` as one of the arguments. 

1. Is my understanding correct that the NMI / AMI metrics get calculated by using the real labels of the `test` split and labels predicted via k-means-clustering on the `test` split? 
2. And as such, information from `samples` also doesn't get used at all?

Thank you in advance!

**Comment by KevinMusgrave** [comment, marked as answer]
Yes to both of your questions
